### ISSUE TITLE
Installing ProDy on a Mac

### ISSUE THREAD

**Issue by AgnieszkaSzarecka** [prody/ProDy#238]
Hi,
I'm trying to install ProDy on my Mac. I have installed Python2.7, and the corresponding Numpy, SciPy and Matplotlib. 
When installing ProDy, i.e. running the "python setup.py build" I get a crash: "Cannot locate working compiler". The full traceback is copied below.

What compiler am I missing and how to get it for MacOX 10.8? Thanks much for help!
Agnieszka

Traceback (most recent call last):
  File "setup.py", line 149, in <module>
    provides=['ProDy ({0:s})'.format(__version__)]
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/command/build.py", line 127, in run
    self.run_command(cmd_name)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/command/build_ext.py", line 307, in run
    customize_compiler(self.compiler)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/sysconfig.py", line 171, in customize_compiler
    _osx_support.customize_compiler(_config_vars)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/_osx_support.py", line 418, in customize_compiler
    _find_appropriate_compiler(_config_vars)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/_osx_support.py", line 191, in _find_appropriate_compiler
    "Cannot locate working compiler")
SystemError: Cannot locate working compiler

**Comment by cihankayacihan** [comment]
Hi. 

Do you have xcode installed? Also, we suggest users to use anaconda python.

**Comment by AgnieszkaSzarecka** [comment]
Hi,
Thanks much for a reply.
I actually managed to solve the problem. Thanks for the xcode tip - that was part of the issue! Strictly speaking, lacking xcode command line tools was a problem. And then ipython needs to installed before ProDy.
I used homebrew and pip step by step to install python and to update my compilers, numpy and scipy. And it worked - in the end :-).  I don't have anaconda, though. But it works fine.
